source/playbooks/notifications-and-updates.rst

```diff
@@ -3,16 +3,20 @@ Notifications and updates
 
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
+  
+.. |vertical-3-dots| image:: ../images/dots-vertical_F01D9.svg
+  :alt: Select the More icon to access additional channel management options.
 
 There are multiple ways to receive updates and notifications.
 
 Status updates
 --------------
 
-Status updates ensure that stakeholders remain informed about the playbook run’s progress. To post a status update:
+Status updates ensure that stakeholders remain informed about the playbook run’s progress. 
 
-1. Select **Toggle Playbook List** from the channel header.
-2. Select **Post update**.
+You can post a status update from the run channel and you can also configure the playbook to send a reminder to post a status update.
```

Comment:
Status updates can be posted from channels RHS and RunDetailsPage. The first paragraph refers to channels RHS and the second paragraph to RunDetailsPage. It's correct, but a bit confusing to me.

Nothing to change if you feel it's ok.